api.py

```diff
@@ -0,0 +1,101 @@
+from flask import Flask, request, jsonify
+
+app = Flask(__name__)   # creates a Flask web application
+
+# In-memory data store (replace with a database in a real-world application)
+users = []
+products = []
+orders = []
+
+# User model
+class User:
+    def __init__(self, id, username, password):
+        self.id = id
+        self.username = username
+        self.password = password
+
+# Product model
+class Product:
+    def __init__(self, id, name, price):
+        self.id = id
+        self.name = name
+        self.price = price
+
+# Order model
+class Order:
+    def __init__(self, id, user_id, product_id, status='Pending'):
+        self.id = id
+        self.user_id = user_id
+        self.product_id = product_id
+        self.status = status
+
+# User registration endpoint

```

Comment:
These type of comments add nothing to the code and just clutter things up. 
What you should do is add detailed descriptions using docstrings for each function as necessary.